app/src/main/java/nextstep/jwp/domain/ContentType.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.jwp.domain;
+
+import java.util.Arrays;
+
+public enum ContentType {
+
+    HTML(".html", "text/html;charset=utf-8"),
+    CSS(".css", "text/css"),
+    JS(".js", "application/javascript"),
+    IMAGE(".svg", "image/svg+xml");
```

Comment:
```suggestion
    IMAGE(".svg", "image/svg+xml"),
    FAVICON(".ico", "image/x-cion");
```

사이트 구성 요소 중 작은 단추 그림인 favicon도 추가해주시는게 좋을 것 같아요!

Reply:
감사합니다 !!